### PR TITLE
Dirty fix to fix brackets without operators

### LINKED ISSUE #1: Does not consider brackets side-by-side as multiplication
See test eq: (1+2)(3+4)+(5+6)(7+8)

Log:
```
Enter your calculation: (1+2)(3+4)+(5+6)(7+8)
[1]
['(', 1.0, '+', 2.0, ')', '(', 3.0, '+', 4.0, ')', '+', '(', 5.0, '+', 6.0, ')', 15.0]
[1]
['(', 1.0, '+', 2.0, ')', '(', 3.0, '+', 4.0, ')', '+', 11.0, 15.0]
[1]
['(', 1.0, '+', 2.0, ')', 7.0, '+', 11.0, 15.0]
[1]
[3.0, 7.0, '+', 11.0, 15.0]
[2]
3.0
```

This was also tested backwards:
```
Enter your calculation: (5+6)(7+8)+(1+2)(3+4)
[1]
['(', 5.0, '+', 6.0, ')', '(', 7.0, '+', 8.0, ')', '+', '(', 1.0, '+', 2.0, ')', 7.0]
[1]
['(', 5.0, '+', 6.0, ')', '(', 7.0, '+', 8.0, ')', '+', 3.0, 7.0]
[1]
['(', 5.0, '+', 6.0, ')', 15.0, '+', 3.0, 7.0]
[1]
[11.0, 15.0, '+', 3.0, 7.0]
[2]
11.0
```

Potential fixes:
- inject a `*` between each bracket set
- return error if no operation seen between elements to calcuate

### PR DESCRIPTION
**Pull request by lifehackerhansol**
This will automatically assume that such case is a multiplication, as per BODMAS rules.

Closes #1.

On a completely unrelated note... would you consider adding an open source license? I'd really like to use this in another project, if you are willing to.

### CODE REVIEW

**Comment by bettercallsean** [comment]
Hello! Sorry, I have no idea how I've only just seen this. Yes, I'll absolutely add an open-source licence for this if you still need to use it, I appreciate you taking the time to add some functionality to this 😄 I'll review it and get back to you ASAP

File: bodmas_calculator.py [hunk 1]
```diff
@@ -11,22 +11,36 @@ def solve(calculation):
...
-
+    bodmas_index = []
+
+    # very dirty hack to check if two values are next to each other with no operators
+    # in this case, this is a multiplication: bracket removals are handled in other functions
+    # so in theory, calculation should be a raw list with no brackets
+    insert_finished = False
+    while not insert_finished:
+        calculation_size = len(calculation)
+        for i in range(len(calculation)):
```
Comment (bettercallsean) [inline bodmas_calculator.py:22]: I believe you can remove line 19, 20, 21, 26 and 27 (basically everything outside the `for` loop), and it should achieve the same result! Will save iterating through the calculation again once the `*` operator has been inserted. Feel free to correct me on this though :)

**Review by bettercallsean — CHANGES_REQUESTED** [review]
Lovely little fix for something that I overlooked when making this! If you could get back to me about the one comment I made that would be great, then I'll get it merged :)

File: bodmas_calculator.py (hunk 1 above)
Comment (lifehackerhansol) [reply bodmas_calculator.py:22]: That does appear to work after some testing. I think I was concerned about it prematurely ending (sanity check if you will.)
Resolved.

**Comment by lifehackerhansol** [comment]
Upon further testing it actually does prematurely end. 
After removing the lines as you suggested this was the result:
```
D:\Documents\Git\BODMAS-Calculator>py bodmas_calculator.py
Enter your calculation: (5+6)(7+8)+(1+2)(3+4)
[1]
['(', 5.0, '+', 6.0, ')', '(', 7.0, '+', 8.0, ')', '+', '(', 1.0, '+', 2.0, ')', 7.0]
[1]
['(', 5.0, '+', 6.0, ')', '(', 7.0, '+', 8.0, ')', '+', 3.0, 7.0]
[1]
['(', 5.0, '+', 6.0, ')', 15.0, '+', 3.0, 7.0]
[1]
[11.0, 15.0, '+', 3.0, 7.0]
[1, 3]
[1]
168.0
```
(The 7.0 was dropped from the last comparison, resulting in `11.0*15.0 + 3.0`)
However the correct result is this:
```
D:\Documents\Git\BODMAS-Calculator>py bodmas_calculator.py
Enter your calculation: (5+6)(7+8)+(1+2)(3+4)
[1]
['(', 5.0, '+', 6.0, ')', '(', 7.0, '+', 8.0, ')', '+', '(', 1.0, '+', 2.0, ')', 7.0]
[1]
['(', 5.0, '+', 6.0, ')', '(', 7.0, '+', 8.0, ')', '+', 3.0, 7.0]
[1]
['(', 5.0, '+', 6.0, ')', 15.0, '+', 3.0, 7.0]
[1]
[11.0, 15.0, '+', 3.0, 7.0]
[1, 5, 3]
[3, 1]
[1]
186.0
```
Compared against a TI-83.
So I reverted the change back to the first commit.

**Comment by bettercallsean** [comment]
Cool that's fine then, thanks for looking into that! Will merge now

**Review by bettercallsean — APPROVED** [review]

**Pull request merged**